app/spike/internal/cmd/policy/get.go

```diff
@@ -64,16 +66,20 @@ import (
 //   - Policy not found
 //   - Insufficient permissions
 //   - JSON formatting failure
-func newPolicyGetCommand(source *workloadapi.X509Source) *cobra.Command {
+func newPolicyGetCommand(
+	source *workloadapi.X509Source, spiffeId string,
+) *cobra.Command {
 	return &cobra.Command{
 		Use:   "get <policy-id>",
 		Short: "Get policy details",
 		Args:  cobra.ExactArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
+			trust.Authenticate(spiffeId)
+
 			api := spike.NewWithSource(source)
 
-			policyID := args[0]
-			policy, err := api.GetPolicy(policyID)
+			policyId := args[0]
```

Comment:
I know golang convention for stuff id is `stuffID`; but most of the code uses `stuffId` so I'm using that to be consistent here too.